beetsplug/replaygain.py

```diff
@@ -181,7 +181,7 @@ def compute_gain(self, items, is_album):
                 i += 1
                 returnchunk = self.compute_chunk_gain(chunk, is_album)
                 albumgaintot += returnchunk[-1].gain
-                albumpeaktot += returnchunk[-1].peak
+                albumpeaktot = max(albumpeaktot, returnchunk[-1].peak)
                 returnchunks = returnchunks + returnchunk[0:-1]
             returnchunks.append(Gain(albumgaintot / i, albumpeaktot / i))
```

Comment:
To compute a maximum, this last `/ i` should be deleted, right?

Reply:
darn.